canonical/vanilla-framework
# Remove no-padding-top from hero signpost title

## Done

Removes `.u-no-padding--top` from the hero signpost titles - this was missed in reviews of the Hero pattern and requested to be fixed [here](https://chat.canonical.com/canonical/pl/1bda8imr5bymbde47unn8hjmoe).

## QA

- Open [signpost hero demo](https://vanilla-framework-5341.demos.haus/docs/examples/patterns/hero/hero-signpost-full-width-image)
- Verify that the `.u-no-padding--top` has been removed from the title text

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix release (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

![image](https://github.com/user-attachments/assets/7062a9ba-6a29-45a4-9aba-29294ca209b7)
